docs/cudf/source/api_docs/series.rst

```diff
@@ -266,8 +266,11 @@ Datetime properties
    days_in_month
    day_of_year
    hour
+   microsecond
```

Comment:
These are sorted alphabetically, but `index_objects.rst` sorts them by size (hour, minute, second). Let's align somehow.

The pandas docs use duration size (not alphabetical order) for both `Series.dt` and `DateTimeIndex` properties:
https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.year.html
https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.year.html

Reply:
Nice catch, will do

Reply:
Done 👍